### DISCUSSION TITLE
*pgx.ConnConfig.BuildStatementCache not longer supported in v5

### DISCUSSION THREAD

**Discussion by sanket28** [jackc/pgx#1535] [Q&A] [answered]
Hello,

In v4 I used to set the *pgx.ConnConfig.BuildStatementCache field to false to disable statement caching. What is the equivalent for this in v5? 

Thanks!

**Comment by jackc** [comment, marked as answer]
Change `DefaultQueryExecMode` in your config. See https://pkg.go.dev/github.com/jackc/pgx/v5#ConnConfig and https://pkg.go.dev/github.com/jackc/pgx/v5#QueryExecMode.
